src/main/java/callhub/connect/controllers/FileController.java

```diff
@@ -0,0 +1,75 @@
+package callhub.connect.controllers;
+import callhub.connect.data_access.DataAccess;
+import callhub.connect.data_access.DocumentRepository;
+import callhub.connect.data_access.LocalDataAccess;
+import callhub.connect.data_access.NetworkDataAccess;
+import callhub.connect.entities.FileDocument;
+import callhub.connect.entities.exceptions.FileLimitExceededException;
+import org.springframework.http.MediaType;
+import org.bson.types.Binary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/files")
+public class FileController {
+
+    @Autowired
+    public DocumentRepository documentRepository;
+    HttpHeaders headers = new HttpHeaders();
```

Comment:
should this be created inside each method?

Reply:
in case we need to add info to the headers in the future

Reply:
I think for headers you customize with methods, the base is always consistent